vcnc-server/vcnc-sampler/app.js

```diff
@@ -87,31 +84,51 @@ function requestHandler(request, response) {
   response.end();
 }
 
-const server = http.createServer(requestHandler);
-server.setTimeout(5000, () => {
-});
+// Create server
+//
+let server;
+function startServer(h, p) {
+  server = http.createServer(requestHandler);
+  server.setTimeout(5000, () => {
+  });
```

Comment:
Line 90:  Why not use [server.timeout(_int_)](https://nodejs.org/api/http.html#http_server_timeout) ??  You're giving an empty callback to server.setTimeout, anyways.

The default for dropping a connection is two minutes.  Are you sure you want to _shorten_ it to 5 seconds?  Why are you not using the default?